assets/scss/base/_helpers.scss

```diff
@@ -373,3 +373,18 @@
   max-width: 125px;
 }
 
+/*
+Equally spaced out container element
+
+Experimental: Equal top and bottom spacing for container element
+
+Markup:
+<div class="{{modifier_class}}">Content</div>
+
+.equal-spacing - equal top and bottom spacing for container element
```

Comment:
You need the `Styleguide` tag to make this show up in the component library https://github.com/hmrc/assets-frontend/wiki/Component-Library#example

in this case `Styleguide: Helpers.Equal Spacing`